APISpecification.yaml

```diff
@@ -0,0 +1,100 @@
+openapi: 3.0.3
+info:
+  title: Ensembl Resolver API
+  description: Resolver API resolves external urls to Ensembl
+  contact:
+    email: ensembl-webteam@ebi.ac.uk
+  version: 0.0.1
+servers:
+  - url: https://resolver.ensembl.org
+tags:
+  - name: resolver
+    description: Resolver API resolves external urls to Ensembl
+paths:
+  /resolve/{gene_stable_id}:
```

Comment:
If it could potentially resolve gene, transcript, variant than name `gene_stable_id` is no appropriate here.

Reply:
The endpoint description suggests `stable_id`

Reply:
just `id`?